ACadSharp/IO/DWG/DwgStreamReaders/DwgObjectReader.cs

```diff
@@ -1048,6 +1054,129 @@ private CadTemplate readUnlistedType(short classNumber)
 			return template;
 		}
 
+		#endregion
+
+		#region Evaluation Graph, Enhanced Block etc.
+
+		private CadTemplate readEvaluationGraph() {
+			EvaluationGraph evaluationGraph = new EvaluationGraph();
+			EvaluationGraphTemplate template = new EvaluationGraphTemplate(evaluationGraph);
+
+			this.readCommonNonEntityData(template);
+
+			//	DXF fields 96, 97 contain the value 5, here are three fields returning the same value 5
+			var val1 = _objectReader.ReadBitLong();
+			var val2 = _objectReader.ReadBitLong();
+			var val3 = _objectReader.ReadBitLong();
+			int nodeCount = val3;
+
+			for (int i = 0; i < nodeCount; i++) {
+				var node = new EvaluationGraph.GraphNode();
+				evaluationGraph.Nodes.Add(node);
+				node.Index = _objectReader.ReadBitLong();
+				node.Flags = _objectReader.ReadBitLong();
+				node.NextNodeIndex = _objectReader.ReadBitLong();
+				template.NodeHandles.Add(node, this.handleReference());
+				node.Data1 = _objectReader.ReadBitLong();
+				node.Data2 = _objectReader.ReadBitLong();
+				node.Data3 = _objectReader.ReadBitLong();
+				node.Data4 = _objectReader.ReadBitLong();
```

Comment:
Here might be an additional ReadBit boolean for active_cycles, when a global computed has_graph is detected.